src/data/messages/messages.json

```diff
@@ -5002,52 +5002,56 @@
                             "id": 5
                         },
                         {
-                            "name": "Capability_EOS",
+                            "name": "Capability_Decred",
                             "id": 6
                         },
                         {
-                            "name": "Capability_Ethereum",
+                            "name": "Capability_EOS",
                             "id": 7
                         },
                         {
-                            "name": "Capability_Lisk",
+                            "name": "Capability_Ethereum",
                             "id": 8
                         },
                         {
-                            "name": "Capability_Monero",
+                            "name": "Capability_Lisk",
                             "id": 9
                         },
                         {
-                            "name": "Capability_NEM",
+                            "name": "Capability_Monero",
                             "id": 10
                         },
                         {
-                            "name": "Capability_Ripple",
+                            "name": "Capability_NEM",
                             "id": 11
                         },
                         {
-                            "name": "Capability_Stellar",
+                            "name": "Capability_Ripple",
                             "id": 12
                         },
                         {
-                            "name": "Capability_Tezos",
+                            "name": "Capability_Stellar",
                             "id": 13
                         },
                         {
-                            "name": "Capability_U2F",
+                            "name": "Capability_Tezos",
                             "id": 14
                         },
                         {
-                            "name": "Capability_Shamir",
+                            "name": "Capability_U2F",
                             "id": 15
                         },
                         {
-                            "name": "Capability_ShamirGroups",
+                            "name": "Capability_Shamir",
                             "id": 16
                         },
                         {
-                            "name": "Capability_PassphraseEntry",
+                            "name": "Capability_ShamirGroups",
                             "id": 17
+                        },
+                        {
+                            "name": "Capability_PassphraseEntry",
+                            "id": 18
```

Comment:
The mappings between message IDs and currencies changed, is this backwards-compatible?